app/serializers/garden_serializer.rb

```diff
@@ -0,0 +1,5 @@
+class GardenSerializer
+  include JSONAPI::Serializer
+  attributes :name, :cardinal_direction, :notes
+  belongs_to :user
```

Comment:
We wont need to have this in this serializer.

It would end up putting info about the User when we just want to see the plant.

Reply:
Okay! I wasn't sure on that part honestly I just remembered something being said about something being passed through in the serialize?